docs/community/100-days-of-oss.md

```diff
@@ -0,0 +1,71 @@
+- write a blog post or create content that supports an OSS project; 
+- update or write documentation to improve clarity and usability
+- create content: give a talk or presentation on OSS, participate in or even start a Twitter Space write a blog post or create a video;  
+- support contributors working on OSS projects, providing guidance and support.
+
+To keep track of your progress, post on social media, your blog, or any other platform you prefer with what progress you made, the day of the challenge indicated by 'D'and include the hashtag #100DaysOfOSS. For example, if you're on day one, you could say, "Today, I reviewed the documentation for the [OpenSauced/Insights](https://github.com/open-sauced/insights) repository. D1 #100daysOfOSS." Then, on day two, you would continue with D2, and so on.
+
+If you're ready to join this challenge, [tweet out your commitment today](https://twitter.com/intent/tweet?text=I%27m%20joining%20the%20%23100DaysOfOSS%20challenge%20with%20%40saucedopen%20to%20grow%20my%20skills%2C%20gain%20real-world%20experience%20in%20open%20source%2C%20and%20to%20make%20a%20positive%20impact%20on%20the%20OSS%20Community!) or share on your platform of choice!
+
+## The Official Kickoff
+We're starting 100 days from the end of [Hacktoberfest](https://hacktoberfest.com/), a month-long celebration of open source contributions. Starting July 23rd, we'll provide continuous support, daily inspirational tweets, and engaging events to help you stay motivated and make progress. 
+
+Don't worry if you're unable to start on the same day as everyone else. The #100DaysOfOSS challenge is flexible, and you can join in whenever you're ready. Just jump in at any point and begin with day one of your personal challenge. 
+
+## Support
+To make the most of your #100DaysOfOSS journey, here are some additional resources and events you can explore:
+
+1. **Weekly Twitter Spaces:** Join our weekly Twitter Spaces sessions where we discuss open source topics, share insights, and connect with like-minded individuals. Follow us on [Twitter](https://twitter.com/saucedopen) to stay updated on upcoming sessions.
+
+2. **Community Events:** Discover a wide range of events on our community docs page. Whether it's hack days, workshops, or office hours, these events provide excellent opportunities to learn, collaborate, and find new projects to contribute to. Visit our [community docs page](https://docs.opensauced.pizza/community/welcome-to-the-community) to explore the upcoming events.
+
+3. **Weekly Contribution Opportunities:** If you're actively looking for open source projects to contribute to, check out the [weekly post](https://dev.to/bekahhw/series/23323) for new contribution opportunities.
+
+4. **Weekly Office Hours:** Have questions or need help? Join us on [Discord](https://discord.gg/dXvGd8mz) during our office hours or post in our #100DaysOfOSS channel. We're here to help you succeed!
+
```

Comment:
[Addition] I think there should be a section on balancing this challenge here. One of the criticisms I heard about 100 Days of Code is that implies this needs to be done for 100 days straight,so putting something like "take breaks" or "reward yourself for doing something OSS-related" would help students to participate.